package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@mondaydotcomorg/storybook-addon-playground",
-  "version": "0.0.1",
+  "name": "storybook-addon-playground",
+  "version": "0.0.0",
```

Comment:
If you publish it this way it should be 0.0.1 at least

Reply:
Or you expecting it to bump it in this pr?

Reply:
I want it to start at .0 and running this workflow once it's merged (I guess before that I will create a branch with dry_run to not create a publish, just to print what if.., and will fix if something needs a fix) will bump to 0.0.1 and commit it and release to NPM the 0.0.1